src/workerd/server/autogate.h

```diff
@@ -12,6 +12,7 @@ namespace workerd::server {
 // Workerd-specific list of autogate keys (can also be used in internal repo).
 enum class AutogateKey {
   TEST_WORKERD,
+  BUILTIN_WASM,
```

Comment:
Nit: let's be sure to get in the habit of documenting these keys when we add them. It'll make it easier for future generations.